CMakeLists.txt

```diff
@@ -1331,8 +1331,13 @@ gtest_add_tests(
   WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
   TEST_LIST testsuite
 )
+
+# Set default test timeout to 30 seconds
```

Comment:
Why don't we simply increase the default timeout?

Reply:
The default timeout is quite big anyway, it'd rather reduce it. Otherwise an endless loop or something like will block the build queue for too long.

Also, a tight timeout can help to discover extremely inefficient code introduced by a PR, because when the test suddenly starts to fail we at least notice it. We do have benchmarks, but they only cover a small part of mixxx and I doubt we'd notice if something starts to take conderably more time.

On my <1000 EUR laptop, most of the tests finish after less than 0.5 seconds.

Reply:
An extended test timeout doesn't hurt because endless loops in tests are very unlikely and will only occur very infrequently. Increasing the total CI build time of ~30-45 min for an additional 15 sec. in rare cases and only when expecting an exceptional failure doesn't matter.

Reply:
In those cases, it would even be acceptable if the CI build fails due to the total CI timeout. The single test timeout actually only matters when running tests locally without a total timeout.

Reply:
OK, I removed all specific test timeouts from CMakeLists to allow using the `--timeout` flag in the CI builds.